lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -363,14 +363,12 @@ def setUp(self):
         (ModuleStoreEnum.Type.split, False, 1, 3, 3, 12, 1),
         (ModuleStoreEnum.Type.split, False, 50, 3, 3, 12, 1),
 
-        # Enabling Enterprise integration increases the number of (cached and uncached) SQL queries by 1,
```

Comment:
Reviewers: this code no longer has an extra query because we don't do the user consent check in JSON calls. It only makes sense to redirect HTML requests which are now handled in the ``CourseTabView``.